testsuite/features/step_definitions/command_steps.rb

```diff
@@ -657,9 +658,17 @@
     next if node.nil?
     node.run("echo -e \"#{conf}\" > /etc/sysconfig/network-scripts/ifcfg-eth1 && systemctl restart network")
   end
+  # /etc/netplan
+  nodes = [$ubuntu_minion]
+  config_file = File.dirname(__FILE__) + '/../upload_files/' + '01-netcfg.yaml'
+  conf = File.readlines(config_file).sub('DOMAINS', domain).sub('ADDRESSES', proxy)
```

Comment:
<s>No. If the file is not uploaded, then it should not be in `../upload_files`.

Just hardcode the data in the test suite. It will be less complicated, easier to maintain, and faster at runtime.

See just above: `conf = "STARTMODE='auto'\\nBOOTPROTO='dhcp'"`. Put your configuration in a `conf` variable, and use interpolation to replace `#{domain}`and `#{proxy}` (faster than calling twice `sub`function).</s>

Ah ok, this file is long. Then okay.